### ISSUE TITLE
Drug Class appears to be inchikey

### ISSUE THREAD

**Issue by andricDu** [docking-org/OICR-ZINC#6]
Example for first drug in drugs.ldjson:

```
"inchikey": "AAKJLRGGTJKAMG-UHFFFAOYSA-N"
...
"drug_class": "AAKJLRGGTJKAMG-UHFFFAOYSA-N
```

**Comment by jir322** [comment]
Thanks. Obviously a bug. We will investigate.

**Comment by teaguesterling** [comment]
I believe this was fixed by 68021906a2690cf4ef79d59f4e7e571600d45a2a. Closing.
